extensions-core/core/runtime/src/main/java/org/apache/camel/quarkus/core/CamelConfig.java

```diff
@@ -62,6 +62,13 @@
         @ConfigItem(defaultValue = "true")
         public boolean enabled;
 
+        /**
+         * If {@code true}, the camel context will be started at initialization time
+         * to boot up faster.
```

Comment:
If there are any situations in which `true` would not work well the JavaDoc should warn the user.

As far as I can understand the option moves camel context start from RUNTIME_INIT to STATIC_INIT. If so, the JavaDoc should not hesitate to use those concepts and should refer to https://quarkus.io/guides/writing-extensions#bootstrap-three-phases for the definitions of the terms.

Reply:
It should not, but I intend to run a PR to check that it can be enabled and still works.  I just need to make all the tests pass before.

Reply:
This is really a first step in the PR.  I'll submit more commits so that we actually have the most interesting part which is to switch the `context.init()` call to the `STATIC_INIT` phase.